app/services/calculators/dependant_allowance_calculator.rb

```diff
@@ -0,0 +1,71 @@
+module Calculators
+  class DependantAllowanceCalculator
+    def self.call(dependant)
+      new(dependant).call
+    end
+
+    def initialize(dependant)
+      @dependant = dependant
+    end
+
+    def call
+      return child_under_15_allowance if under_15_years_old?
+
+      return child_aged_15_allowance - monthly_income if under_16_years_old?
+
+      return child_16_and_over_allowance - monthly_income if under_18_in_full_time_education?
+
+      return 0.0 if capital_over_allowance?
+
+      adult_allowance - @dependant.monthly_income
```

Comment:
`@dependant.monthly_income` here, but `monthly_income` in the two children blocks above

Reply:
Good spot.  Should both be monthly_income